db/migrate/20220509110837_add_id_offset_organisations.rb

```diff
@@ -0,0 +1,7 @@
+class AddIdOffsetOrganisations < ActiveRecord::Migration[7.0]
+  def up
+    execute "SELECT setval('organisations_id_seq', 100000000)"
+  end
+
+  def down; end
```

Comment:
Is there a reason you do this with up/down instead of change?

Reply:
I think making it not reversible is sensible, I don't see what value we could revert to.
Doing it at export is feasible but then we would always have to translate the IDs for export which I find a bit tedious.
We modelled everything so far with the assumption we would match CDS, so let's stay consistent.